### PR TITLE
Added show/hide row

### PR DESCRIPTION
**Pull request by djhvscf**

### CODE REVIEW

**Comment by wenzhixin** [comment]
There are some code format problems, need to update.

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -1415,6 +1415,18 @@
...
+	BootstrapTable.prototype.toggleRow = function (index, visible) {
+		if (index === -1) {
+            return;
+        }
+
+        if (visible) {
+            this.$selectItem.filter(sprintf('[data-index="%s"]', index)).parents('tr').css('display', 'table-row');
+        } else {
+            this.$selectItem.filter(sprintf('[data-index="%s"]', index)).parents('tr').css('display', 'none');
+        }
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: here we can use this maybe better? 
```
this.$selectItem.filter(sprintf('[data-index="%s"]', index)).parents('tr')[visible ? 'show' : 'hide']();
```

File: src/bootstrap-table.js [hunk 2]
```diff
@@ -1415,6 +1415,18 @@
             }
         }
     };
+	
+	BootstrapTable.prototype.toggleRow = function (index, visible) {
+		if (index === -1) {
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: need to replace `tab` to `spaces`.

**Comment by djhvscf** [comment]
Ready

**Comment by wenzhixin** [comment]
Never mind, here maybe can help:
https://www.jetbrains.com/webstorm/help/changing-indentation.html

Settings > Code Style > JavaScript > Tabs and Indents

**Comment by wenzhixin** [comment]
Need to update the docs, pls help.

**Pull request merged**